app/src/Api/GameApi.php

```diff
@@ -26,13 +28,18 @@ public function new(int $size) : Game {
   }
 
   public function move(Game $game, int $tile) : Game {
-    $response = $this->client->post('/move-tile', [
-      'body' => json_encode([
-        'InitialGrid' => $game->getResolvedGrid(),
-        'Grid' => $game->getCurrentGrid(),
-        'TileNumber' => $tile
-      ])
-    ]);
+    try {
+      $response = $this->client->post('/move-tile', [
+        'body' => json_encode([
+          'InitialGrid' => $game->getResolvedGrid(),
+          'Grid' => $game->getCurrentGrid(),
+          'TileNumber' => $tile
+        ])
+      ]);
+    } catch (ConnectException $e) {
+      return $game;
```

Comment:
so the user doesn't see that there was an error?

Reply:
IMO, you should display something like "You can't move that tile. Try another one".